code/modules/shuttles/shuttle.dm

```diff
@@ -134,25 +108,10 @@
 			spawn(0)
 				P.open()
 
-	if (iselevator)	// Super snowflake code
-		for (var/obj/machinery/computer/shuttle_control/ice_colony/C in area)
-			C.animate_off()
-
-		for (var/turf/closed/shuttle/elevator/gears/G in area)
-			G.stop()
-
-		for (var/obj/machinery/door/airlock/D in area)//For elevators
-			if (D.locked)
-				spawn(0)
-					D.unlock()
-			if (D.density)
-				spawn(0)
-					D.open()
-	else
-		for(var/obj/machinery/door/airlock/unpowered/D in area)
-			if(D.density && !D.locked)
-				spawn(0)
-					D.open()
+	for(var/obj/machinery/door/airlock/unpowered/D in area)
+		if(D.density && !D.locked)
+			spawn(0)
```

Comment:
Is the `spawn(0)` necessary?

Reply:
this is gonna get replaced eventually by the shuttle port but in this case yes it does because open/close use a sleep